include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -20,39 +20,41 @@
 
 #pragma once
 
-#if !ENABLE_OPENPMD
-#    error The activated radiation plugin (radiation.param) requires openPMD-api.
-#endif
-
-#include "picongpu/simulation_defines.hpp"
-
-#include "picongpu/particles/traits/SpeciesEligibleForSolver.hpp"
-#include "picongpu/plugins/ISimulationPlugin.hpp"
-#include "picongpu/plugins/common/openPMDDefaultExtension.hpp"
-#include "picongpu/plugins/common/openPMDVersion.def"
-#include "picongpu/plugins/common/stringHelpers.hpp"
-#include "picongpu/plugins/radiation/Radiation.kernel"
-#include "picongpu/plugins/radiation/executeParticleFilter.hpp"
-
-#include <pmacc/dataManagement/DataConnector.hpp>
-#include <pmacc/filesystem.hpp>
-#include <pmacc/lockstep/lockstep.hpp>
-#include <pmacc/math/operation.hpp>
-#include <pmacc/mpi/MPIReduce.hpp>
-#include <pmacc/mpi/reduceMethods/Reduce.hpp>
-#include <pmacc/traits/HasIdentifier.hpp>
-
-#include <algorithm> // std::any
-#include <complex>
-#include <cstdlib>
-#include <fstream>
-#include <iostream>
-#include <memory>
-#include <optional>
-#include <string>
-#include <vector>
-
-#include <openPMD/openPMD.hpp>
+#if(ENABLE_OPENPMD == 1)
+
+#    if !ENABLE_OPENPMD
+#        error The activated radiation plugin (radiation.param) requires openPMD-api.
+#    endif
```

Comment:
```suggestion
```
This looks kind of redundant given the line above.

Reply:
fixed, it was in before too but not visible because the guard and check were in two different files.